app.json

```diff
@@ -36,6 +36,26 @@
     "S3_ASSET_HOST_URL": {
       "description": "Optional custom CDN asset host url, if using S3 file storage.",
       "required": false
+    },
+    "WEBDAV_SERVER_URL": {
+      "description": "URL of your WebDAV server, if using WebDAV file storage",
```

Comment:
From the webdav storage adapter docs, it looks like only this is required if you're using webdav. Would you mind clarifying that if you were to use webdav, only `WEBDAV_SERVER_URL` is mandatory and the other ones are optional (even if you're using webdav)?

Or alternatively, you can just link to the storage adapter docs directly within the README (to which I assume you're adding a "WebDav" section)

Reply:
All done :heavy_check_mark: